src/Plutip/Types.purs

```diff
@@ -36,26 +31,47 @@ import Aeson
   , toStringifiedNumbersJson
   , (.:)
   )
+import Contract.Address
+  ( getWalletAddress
+  , ownPaymentPubKeyHash
+  , ownStakePubKeyHash
+  )
+import Contract.Monad (Contract, liftedE, liftedM, throwContractError)
+import Contract.Prelude (isJust, mconcat)
+import Contract.ScriptLookups as Lookups
+import Contract.Transaction
+  ( TransactionOutput(..)
+  , awaitTxConfirmed
+  , balanceAndSignTxE
+  , signTransaction
+  , submit
+  )
+import Contract.TxConstraints as Constraints
+import Contract.Utxos (utxosAt)
+import Contract.Wallet (withKeyWallet)
 import Data.Array as Array
 import Data.BigInt (BigInt)
 import Data.Either (Either(Left), note)
+import Data.FoldableWithIndex (foldMapWithIndex)
 import Data.Generic.Rep (class Generic)
 import Data.Log.Level (LogLevel)
-import Data.Maybe (Maybe(Just, Nothing))
-import Data.Newtype (class Newtype)
+import Data.Maybe (Maybe(Nothing, Just))
+import Data.Newtype (class Newtype, unwrap, wrap)
 import Data.Show.Generic (genericShow)
 import Data.String as String
 import Data.Tuple (Tuple(Tuple))
 import Data.Tuple.Nested (type (/\), (/\))
 import Data.UInt (UInt)
+import Effect.Class.Console (log)
 import QueryM.ServerConfig (ServerConfig)
 import Serialization (privateKeyFromBytes)
 import Serialization.Types (PrivateKey)
 import Types.ByteArray (hexToByteArray)
 import Types.RawBytes (RawBytes(RawBytes))
 import Wallet.Key
   ( KeyWallet
-  , PrivatePaymentKey(PrivatePaymentKey)
+  , PrivatePaymentKey(..)
```

Comment:
We use explicit imports and exports, including constructors. Please undo these changes here and above

Reply:
Is there an automated way to do this? Emacs is switching to implicit, I'll have to check if it can do the opposite